plugin-build/src/main/kotlin/io/sentry/android/gradle/util/Versions.kt

```diff
@@ -0,0 +1,9 @@
+package io.sentry.android.gradle.util
+
+import com.android.builder.model.Version
+import org.gradle.util.internal.VersionNumber
```

Comment:
Really nit: Not sure we want to depend on `org.gradle.util.internal.` classes.

Reply:
Can we vendor the impl. of VersionNumber within the Plugin?

Reply:
Yeah I was thinking about something like that or re-implement a simple version comparer